resources/views/users/show.blade.php

```diff
@@ -38,12 +40,11 @@
                             <th class="table__cell">Items</th>
                             <th class="table__cell">Quantity</th>
                             <th class="table__cell">Updated At</th>
-                            <th class="table__cell">Flagged</th>
```

Comment:
But I could defs be wrong! @mikefantini do we _need_ the RB flagged stuff in this table view?